tests/ragger/test_name_version.py

```diff
@@ -0,0 +1,14 @@
+from ragger.backend import SpeculosBackend
+
+
+def test_get_app_and_version(cmd):
+    # for now, Speculos always returns app:1.33.7, so this test is not very meaningful
```

Comment:
Can be overruled by env variable `SPECULOS_APPNAME`
https://github.com/LedgerHQ/speculos/blob/b6218801d999ef7777486d691736458adf5e1c07/src/bolos/os.c#L72

Reply:
probably not worth it. Dismissing this remark.